deploy/install.sh

```diff
@@ -22,13 +22,34 @@ envCode=${envCode:-"${1}"}
 location=${location:-"${2}"}
 envTag=${envTag:-"synapse-${envCode}"}
 deploymentName=${3:-"${envTag}-deploy"}
+securityEnabled=${securityEnabled:-false}
+preventDataExfiltration=${preventDataExfiltration:-false}
```

Comment:
We are using a mix of camelcase and all upper case for naming variables. Should we switch to one naming convention to be consistent?

Reply:
bicep is using camelcase variables, and thus we may not want to change those variables. I will update environment variables as CAPITAL then.